packages/contracts/src/libraries/ERC1271.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.20;
+
+/**
+ * @title ERC-1271 Magic Values
+ * @dev Library that defines constants for ERC-1271 related magic values.
+ * @custom:security-contact bounty@safe.global
+ */
+library ERC1271 {
```

Comment:
Nit: Why have a library for ERC1271 with only the constants. Why not move the constants into the email signer or move more 1271 logic into library?